src/main/cms/cms.c

```diff
@@ -1046,9 +1078,11 @@ void cmsUpdate(uint32_t currentTimeUs)
             else if (IS_HI(ROLL)) {
                 key = CMS_KEY_RIGHT;
             }
-            else if (IS_HI(YAW) || IS_LO(YAW))
-            {
-                key = CMS_KEY_ESC;
```

Comment:
This should be probably fixed: https://github.com/betaflight/betaflight/blob/9bae508ab8a7f2d7b9f7a7532aa255e986723dfd/src/main/cms/cms.c#L784 ?

Reply:
No strong opinion, I was just commenting that unrelated code is affected.

Maybe CMS_KEY_PREVIOUS may replace ESC

Reply:
BTW: Is it really wise to add NEXT/ PREVIOUS? It will affect protocols that use `externKey`.
Profile edit may be modal, for example using RIGHT to enter row editing and then UP/DOWN to change value .. 
It may be also possible to remove `osd_profile_cursor` ..